net-mgmt/net-snmp/src/sbin/distro

```diff
@@ -0,0 +1,605 @@
+#!/bin/sh
```

Comment:
I don't want to vet this whole script (especially the license part of it and how it interacts with the bsd license of the plugin in general), but I would suggest moving it to `src/opnsense/scripts/OPNsense/Netsnmp/distro.sh` just to get it out of the standard system path and allow additional syntax checking on it (and I like the fact that it's a POSIX script)

Reply:
Where will this end up in the installed plugin, please? So I can also adapt the path in the snmpd configuration.

Thanks.

Reply:
src = /usr/local

Reply:
Done.